tests_e2e/GuestAgentDcrTestExtension/references

```diff
@@ -0,0 +1 @@
+Utils/
```

Comment:
what is the use of this file?

Reply:
I'm unsure, don't see any references to it anywhere. I'm leaving it in for now with a Todo comment to investigate it's usage.